.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
         - PYTHON=3.5
```

Comment:
Would you mind updating 'conda-build=1.*' on the previous line to `conda-build=2.*'? Think we are safe to drop that one now. ;)

Reply:
I did 2.0.*, as I think the purpose is to test against an older, non-standard build.  2.* would just pick up the current 2.1.x release.

Reply:
That seems fine. Would have been fine with `2.*` as well. Just as long as the version used in c-f is somewhere between the upper and lower bounds ( with the exception of `1.*` of course ;) ).